utility_analysis_new/tests/utility_analysis_test.py

```diff
@@ -136,23 +140,35 @@ def test_w_public_partitions(self, noise_kind, abs_error_expected,
         # Assert
         # Assert a singleton is returned
         self.assertLen(col, 1)
-        # Assert there is only a single AggregateErrorMetrics & no metrics for
+        # Assert there is only a single AggregateMetrics & no metrics for
         # partition selection are returned.
         self.assertLen(col[0], 1)
-        # Assert count metrics are reasonable.
+        # Assert privacy id count metrics are reasonable.
         self.assertAlmostEqual(
-            col[0][0].aggregate_error_metrics.abs_error_expected,
+            col[0][0].privacy_id_count_metrics.abs_error_expected,
```

Comment:
Nit: to avoid duplication
```
def check_metrics(metrics):
   self.assertEqual(abs_error_expected, abs_error_expected)
    ....

check_metric(col[0][0].privacy_id_count_metrics)
check_metric(col[0][0].count_metrics)
```
(AFAIU count and privacy_id count metrics are the same, since each privacy id contributes once, so no per-partition error)